run_cupy_install_test.py

```diff
@@ -41,7 +41,7 @@
     # make sdist
     # cuda, cudnn and numpy is required to make a sdist file.
     build_conf = {
-        'base': 'ubuntu16_py35',
+        'base': 'ubuntu16_py36-pyenv',
```

Comment:
How about using ubuntu18? The EOL of ubuntu16 (2021-04) is coming soon.
```suggestion
        'base': 'ubuntu18_py36',
```

Reply:
Yeah I also tried that, but the problem is that we need to install an older gcc for cuda 9.0 to work :(.
I wonder if we can bump the cuda version for this test? @kmaehashi

Reply:
(internal discussion) We decided to bump cuda version for the install test.